### PR TITLE
"Hotfix" for vessel problems+ minor additions.

### PR DESCRIPTION
**Pull request by Flaksim**
- maptweak: "Added an elevator between deck 2 and MAC controls on the frigate."
  - maptweak: "Added a "supraluminal communications array" (fax) to the Kig-Yar bridge."
  - bugfix: "Fixed deck gun and archer gun controls on the frigate."
  - bugfix: "Fixed Kig-Yar vessel engines and connector port for refueling."
  - experiment: "Changed Sangheili language colour to "vox" entry."

### CODE REVIEW

File: html/changelogs/Flaksim Update 8 june.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - maptweak: "Added an elevator between deck 2 and MAC controls on the frigate."
+  - maptweak: "Added a "supraluminal communications array" (fax) to the Kig-Yar bridge."
```
Comment (bloxgate) [inline html/changelogs/Flaksim Update 8 june.yml]: Replace
```"Added a "supraluminal communications array" (fax) to the Kig-Yar bridge."```
With ```"Added a \"supraluminal communications array\" (fax) to the Kig-Yar bridge."```

File: html/changelogs/Flaksim Update 8 june.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - bugfix: "Fixed deck gun and archer gun controls on the frigate."
+  - bugfix: "Fixed Kig-Yar vessel engines and connector port for refueling."
+  - experiment: "Changed Sangheili language colour to "vox" entry."
```
Comment (bloxgate) [inline html/changelogs/Flaksim Update 8 june.yml]: Replace ```"Changed Sangheili language colour to "vox" entry."``` with ```"Changed Sangheili language colour to \"vox\" entry."```

**Review by bloxgate — CHANGES_REQUESTED** [review]
Changelog has issues

**Review by bloxgate — APPROVED** [review]
Code changes look good. Those that touch things in map still need to be tested in a local environment by another reviewer though.

**Comment by Flaksim** [comment]
So who is going to local test the maps? I did a comprehensive test for a change though, let the kig-yar ship fly for 30 minutes then checked if it still had working engines and refueling, opened and closed the shutter I added, and tested if the new elevator worked.

**Review by Desolane — APPROVED** [review]
Code looks fine.

**Pull request merged**